sdk/typescript/src/providers/json-rpc-provider.ts

```diff
@@ -275,22 +276,22 @@ export class JsonRpcProvider extends Provider {
   ): Promise<GetTxnDigestsResponse> {
     const requests = [
       {
```

Comment:
Do you mind adding a generic `getTransactions` method that exposes the new RPC endpoint(with filtering)?

Reply:
added